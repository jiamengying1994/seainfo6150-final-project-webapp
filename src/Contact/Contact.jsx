import React from 'react';
import style from './Contact.module.css';


const Contact = () => (
  <div className={style.contact_background}>

       <div className={style.contact_text}>
              <div className={style.title}><h1>Contact</h1></div>
              <p>Feel free to contact us.</p >
              <table>
                <tbody>
                  <tr>
                    <td>< img src="./assets/img/contact/email.svg" alt='Email' /></td>
                    <td>Email</td>
                    <td>mygallery@gmail.com</td>
                  </tr>
                  <tr>
                    <td>< img src="./assets/img/contact/phone.svg" alt='Phone' /></td>
                    <td>Phone</td>
                    <td>(206)457-6764</td>
                  </tr>
                  <tr>
                    <td>< img src="./assets/img/contact/address.svg" alt='Address' /></td>
                    <td>Address</td>
                    <td>401 Terry Street, Seattle, WA98109</td>
                  </tr>
                </tbody>
              </table>
            </div>

  </div>
)

export default Contact;
