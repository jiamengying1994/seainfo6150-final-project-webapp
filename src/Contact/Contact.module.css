@media(min-width:300px){
.contact_background {
  height: 100%;
  width: 100%;
  display: flex;
  position: relative;
  margin-bottom: 5rem;
  top:1rem;
  align-items: top;
  justify-content: top;
}

.contact_text{
  font-family: Courier New;
  font-size: 1rem;
  padding-left: 5%;
  padding-bottom: 5%;
  background-color:hsl(0, 0%, 90%, 0.5);
  width: 90%;
  height: 100%;
}
.title{
  font-size: 1rem;
  font-family: Courier New;
}

.contact_text img{
  width: 20px;
  height: 10px;}

}

@media(min-width:600px){

.contact_background {
  height: 100%;
  width: 100%;
  display: flex;
  position: relative;
  margin-bottom: 5rem;
  top:1rem;
  align-items: top;
  justify-content: top;
}

.contact_text{
  font-family: Courier New;
  font-size: 1rem;
  padding-left: 5%;
  padding-bottom: 5%;
  background-color:hsl(0, 0%, 90%, 0.5);
  width: 90%;
  height: 100%;
}
.title{
  font-size: 1.5rem;
  font-family: Courier New;
}

.contact_text img{
  width: 30px;
  height: 20px;}

}

@media(min-width:900px){
.contact_background {
  height: 100%;
  width: 100%;
  display: flex;
  position: relative;
  margin-bottom: 5rem;
  top:1rem;
  align-items: top;
  justify-content: top;
}

.title{
  font-size: 2rem;
  font-family: Courier New;
}
.contact_text{
  font-family: Courier New;
  font-size: 2rem;
  padding-left: 5%;
  padding-bottom: 5%;
  background-color:hsl(0, 0%, 90%, 0.5);
  width: 90%;
  height: 100%;
}

.contact_text img{
  width: 40px;
  height: 30px;
}}








