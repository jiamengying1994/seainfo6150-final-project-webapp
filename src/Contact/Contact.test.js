import React from "react";
import { render } from "@testing-library/react";
import Contact from "./Contact";

it("renders the contact component correctly", () => {
  const { container } = render(<Contact />);
  expect(container).toMatchSnapshot();
});