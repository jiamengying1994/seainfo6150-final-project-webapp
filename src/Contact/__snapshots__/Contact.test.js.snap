// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the contact component correctly 1`] = `
<div>
  <div
    class="contact_background"
  >
    <div
      class="contact_text"
    >
      <div
        class="title"
      >
        <h1>
          Contact
        </h1>
      </div>
      <p>
        Feel free to contact us.
      </p>
      <table>
        <tbody>
          <tr>
            <td>
              <img
                alt="Email"
                src="./assets/img/contact/email.svg"
              />
            </td>
            <td>
              Email
            </td>
            <td>
              mygallery@gmail.com
            </td>
          </tr>
          <tr>
            <td>
              <img
                alt="Phone"
                src="./assets/img/contact/phone.svg"
              />
            </td>
            <td>
              Phone
            </td>
            <td>
              (206)457-6764
            </td>
          </tr>
          <tr>
            <td>
              <img
                alt="Address"
                src="./assets/img/contact/address.svg"
              />
            </td>
            <td>
              Address
            </td>
            <td>
              401 Terry Street, Seattle, WA98109
            </td>
          </tr>
        </tbody>
      </table>
    </div>
  </div>
</div>
`;
